### PR TITLE
Restyle metric field per UX

### PR DESCRIPTION
**Pull request by amattu2**
### Overview

Minor UX updates to the Operation Dashboard metric dropdown.

### Change Details (Specifics)

- Restyle the Operation Dashboard page to remove negative margins for positioning
- Adjust the dropdown and label styling per UX
- Fix the QuickSight frame not rendering the rounded border

### Related Ticket(s)

CRDCDH-1428 (Design task)

### CODE REVIEW

File: src/content/operationDashboard/DashboardView.tsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import {
   FrameOptions,
 } from "amazon-quicksight-embedding-sdk";
 import StyledSelect from "../../components/StyledFormComponents/StyledSelect";
```
Comment (Alejandro-Vega) [inline src/content/operationDashboard/DashboardView.tsx:11]: There are some weird overflowing corners after you click the dropdown then click away to close the dropdown and the input is still in focus.

Screenshot 2024-11-08 at 9 45 16 AM

I think I found the cause, this seems to fix it. Could you add something like this directly to the StyledSelect component?

```javascript
"&.Mui-focused .MuiOutlinedInput-input:focus": {
    borderRadius: "8px",
},
```

File: src/content/operationDashboard/DashboardView.tsx [hunk 2]
```diff
@@ -20,45 +21,42 @@ export type DashboardViewProps = {
   loading: boolean;
 };
 
-const StyledViewHeader = styled(Box)({
+const StyledPageContainer = styled(Box)({
   background: `url(${bannerSvg})`,
```
Comment (Alejandro-Vega) [inline src/content/operationDashboard/DashboardView.tsx:25]: The background in the Figma seems to be different. Are we going with the one you are currently using?

**Review by Alejandro-Vega — CHANGES_REQUESTED** [review]
Mostly LGTM! One minor change and one question.

File: src/content/operationDashboard/DashboardView.tsx (hunk 1 above)
Comment (amattu2) [reply src/content/operationDashboard/DashboardView.tsx:11]: That's some really odd behavior. Thanks for catching it, fixed in b8cd159.

File: src/content/operationDashboard/DashboardView.tsx (hunk 2 above)
Comment (amattu2) [reply src/content/operationDashboard/DashboardView.tsx:25]: TBD, but it doesn't seem like it. If I get an answer, I'll update it in the future.

**Review by Alejandro-Vega — APPROVED** [review]
Didn't re-test but code LGTM!

**Pull request merged**